<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="LAPTOP-JEI8AS0O" name="testcases.HRMTestCases" tests="5" failures="0" timestamp="15 Dec 2019 19:54:53 GMT" time="18.921" errors="1">
  <testcase name="verifyLogin" time="4.779" classname="testcases.HRMTestCases"/>
  <testcase name="validateFirstLevelMenu" time="5.180" classname="testcases.HRMTestCases"/>
  <testcase name="second" time="3.759" classname="testcases.HRMTestCases"/>
  <testcase name="fisrt" time="5.203" classname="testcases.HRMTestCases">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method testcases.HRMTestCases.fisrt() didn&#039;t finish within the time-out 2">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testcases.HRMTestCases.fisrt() didn't finish within the time-out 2
at java.lang.Thread.interrupt(Thread.java:919)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:64)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- fisrt -->
  <testcase name="third" classname="testcases.HRMTestCases">
    <skipped/>
  </testcase> <!-- third -->
</testsuite> <!-- testcases.HRMTestCases -->
